com.unity.render-pipelines.core/Editor/Lighting/LightAnchorEditor.cs

```diff
@@ -266,7 +268,7 @@ public override void OnInspectorGUI()
                 {
                     Undo.RecordObjects(new UnityEngine.Object[] { target, manipulator.transform }, "Light Anchor Change");
 
-                    manipulator.frameSpace = m_FrameSpace;
+                    manipulator.frameSpace = (LightAnchor.UpDirection)m_FrameSpaceProperty.enumValueIndex;
```

Comment:
Check HDEditorUtils.GetEnumValue for details
Never use enumValueIndex to convert to type!
```suggestion
                    manipulator.frameSpace = (LightAnchor.UpDirection)m_FrameSpaceProperty.intValue;
```